rust/kcl-lib/std/solid.kcl

```diff
@@ -147,6 +192,12 @@ export fn chamfer(
   length: number(Length),
   /// The paths you want to chamfer
   tags: [Edge; 1+],
+  /// Use a second length for the second face. Incompatible with `angle`
+  secondLength?: number(Length),
+  /// The angle between the edges. Incompatible with `secondLength`
```

Comment:
We should include the valid range of the angle in the docs of the parameter here so that TTC can produce the right code without executing it. This has been a common mistake we've made that ends up in a bug report. So I've been trying to get better about doing this right from the start.

You should be able to regen docs using `just redo-kcl-stdlib-docs-no-imgs`. Just wanted to make sure you knew about the `no-imgs` version that's much faster.